src/Card/DetailCard/DetailCard.stories.tsx

```diff
@@ -35,8 +35,6 @@ export const Default = {
     content: null,
     labels: [],
     media: "",
-    mediaHeight: 200,
-    mediaWidth: 400,
     onClickDownload: () => {},
```

Comment:
I think we should have a story with version chip or include verison chip in the scenario example story. I think for doing this yon need to have a verion prop

Reply:
You are correct, for now it is required for `SummaryCard` only.